presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java

```diff
@@ -53,7 +53,7 @@
     public TestTpchDistributedStats()
             throws Exception
     {
-        runner = createQueryRunnerWithoutCatalogs(emptyMap(), emptyMap());
+        runner = createQueryRunnerWithoutCatalogs(emptyMap(), emptyMap(), 4);
```

Comment:
Can you just add another overloaded method that does not take a worker count (similar to what you did already)?